File: 0342-power-of-four/0342-power-of-four.py
class Solution(object):
    def isPowerOfFour(self, n):
        """
        :type n: int
        :rtype: bool
        """
        i = 0
        while (i < n):
            if 4**i < n:
                i += 1
            elif 4**i == n:
                return True
            else:
                return False

File: 0342-power-of-four/0342-power-of-four.java
class Solution {
    public boolean isPowerOfFour(int n) {
        int i = 0;
        /*if (Math.pow(4,i) < n){
            i++;
        }
        else if (Math.pow(4,i) == n){
            return true;
        }
        return false;*/


        while (Math.pow(4,i) < n){
            i++;
        }

        if (Math.pow(4,i) == n){
            return true;
        }
        else{
            return false;
        }
    }
}